module/motion/IKKick/src/IKKick.cpp

```diff
@@ -150,9 +150,9 @@ namespace module::motion {
                 Eigen::Vector3d directionSupportFoot = torsoPose.rotation() * directionTorso;
 
                 Eigen::Vector3d ballPosition = targetSupportFoot;
-                ballPosition.z()             = 0.05;  // TODO: figure out why ball height is unreliable
```

Comment:
Change this to "TODO: get ball height from config" (I think we want FieldDescription for this??)